app/views/components/gds/siteHeader.scala.html

```diff
@@ -14,17 +14,31 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukHeader
-@import uk.gov.hmrc.govukfrontend.views.viewmodels.header.Header
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcHeader
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.Header
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language
 
-@this(govukHeader: GovukHeader)
+@this(hmrcHeader: HmrcHeader)
 
-@()(implicit request: Request[_], messages: Messages)
+@(showSignOut: Boolean = true)(implicit messages: Messages)
 
-@govukHeader(Header(
-    homepageUrl = None,
+@languageSelected = @{ if(messages.lang.code == "cy")
```

Comment:
Are we supporting Welsh now?

Reply:
As @PauloPortugal told me, we are going to implement it in the next ~3 months. I'm not going to be an advocate to keep it now, but I'd like to know what everyone thinks about it.

Reply:
What happens if you select Welsh?  Does it default to English?

Reply:
It seems so.

Reply:
That's good.  Do we have an opinion from Product Team about putting this feature in if it doesn't fully work?